src/main/resources/i18n/Messages_en.properties

```diff
@@ -120,21 +120,6 @@ notification.accession.dryingEnd.email.subject=An accession has dried.
 notification.accession.email.buttonIntro=Please click the button below to go to your Terraware account where you can view the accession.
 # Notifications.
 notification.accession.email.linkIntro=Please click the link below to go to your Terraware account where you can view the accession.
-notification.application.accepted.app.body=Your Application was accepted\! Please look out for more details on next steps.
```

Comment:
These will be added back by Phrase unless we go to their web UI and remove the strings from the project after this is merged, which we can definitely do.